content/en/docs/demos/global_rate_limit_connections.md

```diff
@@ -0,0 +1,327 @@
+    20:41:47 I periodic.go:723> T002 ended after 44.0273ms : 3 calls. qps=68.13954069406937
+    Ended after 44.2097ms : 10 calls. qps=226.19
+    Aggregated Function Time : count 10 avg 0.01096615 +/- 0.01386 min 0.001588 max 0.0386716 sum 0.1096615
+    # range, mid point, percentile, count
+    >= 0.001588 <= 0.002 , 0.001794 , 40.00, 4
+    > 0.002 <= 0.003 , 0.0025 , 60.00, 2
+    > 0.003 <= 0.004 , 0.0035 , 70.00, 1
+    > 0.025 <= 0.03 , 0.0275 , 90.00, 2
+    > 0.035 <= 0.0386716 , 0.0368358 , 100.00, 1
+    # target 50% 0.0025
+    # target 75% 0.02625
+    # target 90% 0.03
+    # target 99% 0.0383044
+    # target 99.9% 0.0386349
+    Error cases : no data
+    Sockets used: 3 (for perfect no error run, would be 3)
+    Total Bytes sent: 240, received: 240
+    tcp OK : 10 (100.0 %)
+    All done 10 calls (plus 0 warmup) 10.966 ms avg, 226.2 qps
+    ```
+
+    As seen above, all the TCP connections from the `fortio-client` pod succeeded.
+    ```
+    Total Bytes sent: 240, received: 240
+    tcp OK : 10 (100.0 %)
+    All done 10 calls (plus 0 warmup) 10.966 ms avg, 226.2 qps
+    ```
+
+1. Next, apply a global rate limiting policy to rate limit L4 TCP connections to the `fortio.demo.svc.cluster.local` service that generate the descriptor entry `my_key: my_value` for the `test` domain via the global `ratelimiter.rls.svc.cluster.local` RLS service listening on port `8081`. The descriptor entries and domain correspond to the rate limit configuration specified in the `ratelimit-config` ConfigMap previously. This has the utlimate effect of rate limiting TCP connections inbound on the `fortio` service to 1 per minute by generating the descriptor entry that matches the one specified in the `ratelimit-config` ConfigMap.
+    ```bash
+    kubectl apply -f - <<EOF
+    apiVersion: policy.openservicemesh.io/v1alpha1
+    kind: UpstreamTrafficSetting
+    metadata:
+      name: fortio-tcp
+      namespace: demo
+    spec:
+      host: fortio.demo.svc.cluster.local
+      rateLimit:
+        global:
+          tcp:
+            rateLimitService:
+              host: ratelimiter.rls.svc.cluster.local
+              port: 8081
+            domain: test
+            failOpen: false
+            timeout: 10s
+            descriptors:
+              - entries:
+                - key: my_key
+                  value: my_value
+    EOF
+    ```
+
+    The above configuration will result in connections to the `fortio` backend service to generate the descriptor entry `my_key: my_value` that is then forwarded to the RLS service for making a rate limit decision.
+
+    Confirm no traffic has been rate limited yet by examining the stats on the `fortio` backend pod.
+    ```console
+    $ fortio_server="$(kubectl get pod -n demo -l app=fortio -o jsonpath='{.items[0].metadata.name}')"
+
+    $ osm proxy get stats "$fortio_server" -n demo | grep ratelimit.*fortio.*8078
```

Comment:
Side note: do we have docs on this command? It seems really useful

Reply:
I don't think we have a generic guide on useful CLI commands, so this is likely going to be found in some troubleshooting or demo guides.

Reply:
I'll make an issue to improve our CLI docs; this kind of proxy stat command would be really appealing to a customer